index.js

```diff
@@ -10,4 +11,4 @@ import Modal        from './src/components/modal'
 // XXX: backward compat
 let ActionBar = Navbar
```

Comment:
backward compatibility,
you should probably remove that now.
we use to call ActionBar Navbar, and the rest of the code depended on it.
now that you have a real navbar you should probably remove that hack.